submissions/ToyinAyorinde/Html/Product/product.js

```diff
@@ -0,0 +1,39 @@
+let products = [
+    {
+        name:"Two Diamond and Turquoise Circle Pendant" ,
+        imageUrl: "../images/Product1.jpg",
+        price:"$2,000"
+
+    },
+    {
+        name:"Two Diamond and Mother-of-pearl Circle Pendant" ,
+        imageUrl: "../images/Product2.jpg",
+        price:"$2350"
+
+    },
+    {
+        name:"Two Diamond and Turquoise Circle Pendant" ,
+        imageUrl: "../images/Product3.jpg",
+        price:"$2,550"
+
+    }
+
+]
+
+let imageDiv = document.getElementsByClassName('prdt-img');
+console.log(imageDiv);
+let productName = document.getElementsByClassName('product-name');
+console.log(productName);
+let productPrice = document.getElementsByClassName('product-price');
+console.log(productPrice);
+let button  = document.getElementById("btn");
+console.log(button);
+button.addEventListener('click', displayProduct);
+
+function displayProduct() {
```

Comment:
nit: `displayProducts`, since you are displaying more than one product

Reply:
Done